src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -63,12 +60,6 @@ public void Enable()
                         return;
                     }
 
-                    if (!FeaturesEnabled())
```

Comment:
Note:  6 lines above this is:

```
                     if (!_workspace.Options.GetOption(RemoteHostOptions.RemoteHost))
                     {
                         // not turned on
                          return;
                      }
```

This is the master control switch for OOP.  All this PR does is remove the broken 'feature' level checking.

1. The feature level checking was checking a C# option in a shared package location.  This can be very bad as it means loading the C# dlls even for VB or TS scenarios that don't need it.
2. The feature level checking was only happening at package load time.  That meant that if the feature was off, but was enabled by the user, it would run in proc.
3. The feature level testing only knew about CodeLens and C# FSA.  It didn't know about FAR or NavigateTo.  But there is no on/off option for those features. They're features that users explicitly invoke and are 'always available'.  As such, even if we wanted this stuff to run OOP (i.e. we turned on 'RemoteHostOptions.RemoteHost'), they only would if CodeLens or C#-FSA was enabled.  

This change makes no decisions about OOP at all.  It just makes it so that if OOP is available, that these features can use it.  Unlike the current situation where OOP may be turned on, but is then disabled because CodeLens or FSA isn't on.